.github/workflows/transformer-docker-publish.yml

```diff
@@ -27,15 +27,18 @@ jobs:
       - name: Checkout source
         uses: actions/checkout@v4
 
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
       - name: Run tests
-        run: |
-          if [ -f docker-compose.test.yml ]; then
-            docker-compose --file docker-compose.test.yml build
-            docker-compose --file docker-compose.test.yml run sut
-          else
-            cd python
-            docker buildx build . --file custom_transformer.Dockerfile
-          fi
+        uses: docker/build-push-action@v5
+        with:
+          platforms: linux/amd64
```

Comment:
some places there still only amd64, is it due compatibility with dependencies for arm?

Reply:
That is for custom model and transformer which is only servers as an example. We will have to add ARM support for huggingface in a separate PR.

Reply:
Any issue with the ARM build for huggingface server?

Reply:
Yeah
```
#8 0.202 exec /bin/sh: exec format error
#8 ERROR: process "/bin/sh -c apt-get update -y && apt-get install python3.10-venv -y && apt-get clean &&     rm -rf /var/lib/apt/lists/*" did not complete successfully: exit code: 1

#5 [builder  1/13] FROM docker.io/nvidia/cuda:12.1.0-devel-ubuntu22.04@sha256:e3a8f7b933e77ecee74731198a2a5483e965b585cea2660675cf4bb152237e9b
#5 CANCELED
------
 > [prod 2/7] RUN apt-get update -y && apt-get install python3.10-venv -y && apt-get clean &&     rm -rf /var/lib/apt/lists/*:
0.202 exec /bin/sh: exec format error
------
huggingface_server.Dockerfile:39

```

Reply:
nvm, my docker builder is not configured properly.

Reply:
Looks like vllm does not provides arm64 wheels